### DISCUSSION TITLE
is it possible to batch queue a list of prompts (automatically)?

### DISCUSSION THREAD

**Discussion by dave8x** [Acly/krita-ai-diffusion#1128] [Q&A]
ive been trying to search for it but didnt find anything related. I have a list of prompts which i wish to queue and let krita generate on its own overnight. I know i can copy and paste each prompt and add to queue one by one, but i have a long list and i wish to confirm theres no easier option before doing so... maybe like seperating each prompt via a command or string of characters or something
thanks in advance

**Comment by Acly** [comment]
No, it's probably not hard with a bit of scripting, but there is no UI for something like that.

**Comment by jfuentesa** [comment]
The main prompt window could be used to specify the prompts list, like this:
1. Prompt 1
2. Prompt 2
and so...

**Comment by ETCore7** [comment]
> ive been trying to search for it but didnt find anything related. I have a list of prompts which i wish to queue and let krita generate on its own overnight. I know i can copy and paste each prompt and add to queue one by one, but i have a long list and i wish to confirm theres no easier option before doing so... maybe like seperating each prompt via a command or string of characters or something thanks in advance

If it is pure prompt generation... The plugin is an interface between Krita and Comfy. In case you want pure prompt creation you can skip the Krita/plugin part and batch generate images from within Comfy itself. Your request is generic, there most probably are Comfy plugins/scripts you can install via the Comfy UI.

**Comment by OC-LUIS** [comment]
The most logical way to do this would be to create a list of events each time the generate, refine, etc. button is pressed. This event would consist of a line where the characteristics of the event would appear: generation, deletion, adding content, etc., the style, a summary of the PROMPT, and the images it is going to generate.
Once the list is created and running, one should be able to change the characteristics of pending jobs, especially the number of images per job and be able to set it to 0. Or change the style on the fly with a dropdown.
